CMSgov/mint-app
# [EASI-2424] Sample Model Plan Help Article

# EASI-2424

## Changes and Description

- Add `sample-model-plan` to the `scrollBlackList` and refactored the way it checks if path is included 
- Add new prop `isHelpArticle` to CRTDLs Tables and Document Tables
- Add new prop `isHelpArticle` to SideNav and MobileNav to filter out `Discussions` from the list 
- Add `helpRoute` if `isHelpArticle` is true in ReadOnly.tsx
- Add a Sample Model Plan constants file 
- a bunch of BE stuff that @StevenWadeOddball put so I (@garyjzhao ) 🤷🏼 
- Add tag to `ArticleCard` component

## Acceptance Criteria

- Query a specific model plan (see notes) to populate readonly data
- Include 'Related Help Articles' component underneath readonly (doesn't need to render
directly within readonly component)
- Hide discussions sub nav element
- `<h2>` styling instead of `<h1>` for header
- Include task bar side-nav link that open in a new tab

## How to test this change

1. Navigate to [`the Sample Model Plan`](http://localhost:3005/help-and-knowledge/sample-model-plan)
6. Ensure page is rendering properly (Not 404 error in any way)
7. Page should match figma
8. Note that `Discussions` is not in the side nav
9. Documents and CRs and TDLs should not have any `Edit` button showing, regardless Assessment or Collaborator status


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
